.gitignore

```diff
@@ -9,11 +9,15 @@ target/
 .idea/
 
 # Ignore [ce]tags files
-tags
+/tags
 
 # Metals
 .metals/
 .bsp/
 .bloop/
 metals.sbt
 .vscode
+
+# Generated STD Code
+calico/src/main/scala/calico/html/defs
```

Comment:
Generated code should go in `sourceManaged`.
https://www.scala-sbt.org/1.x/docs/Howto-Generating-Files.html

Reply:
Hmm:
```

Reply:
Hm, classpath issue is left, scalac doesn't see the classes from sources:
```
[error] 19 |import calico.html.keys.AriaAttr
[error]    |       ^^^^^^^^^^^^^^^^
[error]    |     value keys is not a member of calico.html - did you mean html.defs
```

Reply:
seems to be a cyclic error, looks like sbt wants to compile sourceManaged in an isolated run. Maybe this can be solved by introducing another sbt subproject. Will try tomorrow.

Reply:
Huh, that seems strange. Pretty sure I've used `sourceManaged` with cycles before.

Did you push these changes? I can take a quick look.

Reply:
Pushed now